From 81063c796c109393f6868f335bf2a61423ee436b Mon Sep 17 00:00:00 2001
From: YuChia Kan <raspberrykan@stu.hqu.edu.cn>
Date: Mon, 31 Aug 2026 18:11:25 +0800
Subject: [PATCH 6/6] emmctool: allow Xiaomi Mi Box 4 image writes

---
 packages/tools/emmctool/scripts/emmctool | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/packages/tools/emmctool/scripts/emmctool b/packages/tools/emmctool/scripts/emmctool
index cd76b47..73e990d 100755
--- a/packages/tools/emmctool/scripts/emmctool
+++ b/packages/tools/emmctool/scripts/emmctool
@@ -207,7 +207,7 @@ if [ -z "${EMMC}" ]; then
 fi
 
 case $(dtname) in
-  azw*|bananapi*|friendlyarm*|hardkernel*|khadas*|libretech*|radxa*|wetek*)
+  azw*|bananapi*|friendlyarm*|hardkernel*|khadas*|libretech*|radxa*|wetek*|xiaomi,mibox4)
     if [ "${BOOT}" = "${EMMC}" ]; then
       do_info
       echo "Your device is booted from the eMMC module!"
-- 
2.55.0.windows.3

